go/cli/mcap/cmd/doctor.go

```diff
@@ -140,8 +140,12 @@ func (doctor *mcapDoctor) examineChunk(chunk *mcap.Chunk) {
 				doctor.error("Failed to parse schema:", err)
 			}
 
-			if schema.Encoding == "" && len(schema.Data) > 0 {
-				doctor.error("Schema.data field should not be set when Schema.encoding is empty")
+			if schema.Encoding == "" {
```

Comment:
this is a missed fix from #700 - hope reviewers don't mind me merging in this PR.